_data/simple-icons.json

```
@@ -14895,6 +14895,11 @@
             "hex": "3B5526",
             "source": "https://github.com/sympy/sympy.github.com/blob/e606a6dc2ee90b1ddaa9c36be6c92392ab300f72/media/sympy-notailtext.svg"
         },
+        {
+            "title": "Syncthing",
+            "hex": "179CD2",
```

`#0891D1` is used for the [wordmark ](https://syncthing.net/img/logo-horizontal.svg) from their [website](https://syncthing.net/) header. It is also the blue color used in their [forum](https://forum.syncthing.net/) stylesheet.
```suggestion
            "hex": "0891D1",
```